index.html

```diff
@@ -505,5 +507,16 @@ <h2>Results</h2>
         </div>
       </div>
     </main>
+    <script>
+      function changeDisplay(){
+        var box = document.getElementById("gradient-border-input")
+        var chkBox = document.getElementById("gradient-border-radius");
+        if(chkBox.checked == true){
+          box.setAttribute("style", "display: inline");
+        }else if(chkBox.checked == false){
+          box.setAttribute("style", "display: none");
+        }
+      }
+    </script>
```

Comment:
Is it really necessary to use a script tag in the HTML to fix this bug?
- Consider implementing this logic in the appropriate component (main.ts, gradient-border.ts etc)
- Secondly, it's not recommended to use `var` for declaring variables, consider using `let` or `const` for all variable declarations in future